documentation-site/components/yard/compiler.tsx

```diff
@@ -106,7 +106,9 @@ const Compiler: React.FC<{
           justifyContent: 'center',
         })}
       >
-        {Element ? <Element /> : <PlaceholderElement />}
+        <div className={css({width: '400px'})}>
```

Comment:
Adding another wrapper here will remove horizontal centering for smaller components. Could you add this style to the existing outer wrapper? What is this width restriction for?

Reply:
Done. This would be otherwise breaking button's centering.